src/haconiwa/scan/scanner.py

```
@@ -0,0 +1,342 @@
+        """Get information about a file"""
+        info = {
+            'path': str(file_path.relative_to(self.base_path)),
+            'name': file_path.name,
+            'type': self.file_type_mappings.get(file_path.suffix, 'other'),
+            'size': file_path.stat().st_size
+        }
+        
+        if include_content and info['size'] < 1024 * 1024:  # Max 1MB
+            try:
+                info['content'] = file_path.read_text(encoding='utf-8', errors='ignore')
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)